spk/python311-wheels/Makefile

```diff
@@ -1,4 +1,4 @@
-SPK_NAME = python-wheels
+SPK_NAME = python311-wheels
```

Comment:
sorry but this breaks the code in  prepare.sh to move the python* packages to the head of the list

Reply:
@th0ma7 I have a fix for this, no need to rename the python311-wheels package...